source/integrations/net-promoter-score.rst

```diff
@@ -0,0 +1,72 @@
+Net Promoter Score Plugin
+=========================
+
+Mattermost is introducing in-product feedback surveys beginning in v5.12. Feedback is collected in the form of a Net Promoter Score survey, and is used to measure user satisfaction and improve product quality by hearing directly from users. 
+
+.. contents::
+  :depth: 2
+  :local:
+  :backlinks: entry
+
+Administration
+--------------
+Is the survey enabled by default?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Net Promoter Score (NPS) plugin and surveys are enabled by default on all servers that have `Error and Diagnostic Reporting <https://docs.mattermost.com/administration/telemetry.html>`_ enabled when upgrading to v5.12 or later. 
+
+How can surveys be disabled?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Removing or disabling the **Net Promoter Score** plugin from **System Console > Plugins (Beta) > Plugin Management** will disable surveys. Alternatively, you can disable surveys from the plugin configuration in **System Console > Plugins (Beta) > Net Promoter Score**. 
```

Comment:
We could add a note about why one would disable surveys from the plugin configuration.

> If surveys have been disabled from the plugin configuration, users can still send written feedback by messaging Surveybot.